cpp/include/cudf/legacy/io_readers.hpp

```diff
@@ -191,45 +198,98 @@ enum quote_style {
-  bool          delim_whitespace = false;   ///< Use white space as the delimiter; default is false. This overrides the delimiter argument.
-  bool          skipinitialspace = false;   ///< Skip white spaces after the delimiter; default is false.
-  bool          skip_blank_lines = true;    ///< Indicates whether to ignore empty lines, or parse and interpret values as NA. Default value is true.
-  cudf::size_type header = 0;                 ///< Row of the header data, zero based counting; Default is zero.
-
-  std::vector<std::string> names;           ///< Ordered List of column names; Empty by default.
-  std::vector<std::string> dtype;           ///< Ordered List of data types; Empty by default.
-
-  std::vector<int> use_cols_indexes;        ///< Indexes of columns to be processed and returned; Empty by default - process all columns.
-  std::vector<std::string> use_cols_names;  ///< Names of columns to be processed and returned; Empty by default - process all columns.
-
-  std::vector<int> infer_date_indexes;      ///< Column indexes to attempt to infer as date
-  std::vector<std::string> infer_date_names;///< Column names to attempt to infer as date
-
-  std::vector<std::string> true_values;     ///< List of values to recognize as boolean True; Empty by default.
-  std::vector<std::string> false_values;    ///< List of values to recognize as boolean False; Empty by default.
-  std::vector<std::string> na_values;       /**< Array of strings that should be considered as NA. By default the following values are interpreted as NA: 
-                                            '', '#N/A', '#N/A N/A', '#NA', '-1.#IND', '-1.#QNAN', '-NaN', '-nan', '1.#IND', '1.#QNAN', 'N/A', 'NA', 'NULL',
-                                            'NaN', 'n/a', 'nan', 'null'. */
-
-  bool          keep_default_na = true;     ///< Keep the default NA values; true by default.
-  bool          na_filter = true;           ///< Detect missing values (empty strings and the values in na_values); true by default. Passing false can improve performance.
-
-  std::string   prefix;                     ///< If there is no header or names, prepend this to the column ID as the name; Default value is an empty string.
-  bool          mangle_dupe_cols = true;    ///< If true, duplicate columns get a suffix. If false, data will be overwritten if there are columns with duplicate names; true by default.
-
-  char          quotechar = '\"';           ///< Define the character used to denote start and end of a quoted item; default is '\"'.
-  quote_style   quoting = QUOTE_MINIMAL;    ///< Defines reader's quoting behavior; default is QUOTE_MINIMAL.
-  bool          doublequote = true;         ///< Indicates whether to interpret two consecutive quotechar inside a field as a single quotechar; true by default.
-
-  gdf_time_unit out_time_unit = TIME_UNIT_NONE; ///< Defines the output resolution for date32, date64, and timestamp columns
+  std::string compression =
```

Comment:
Looks like the nice aligned formatting was replaced with something different?

Reply:
hmmm? Looks like my Google formatter in vscode did that. I will change that back.